packages/rx-effects-react/src/useObserver.ts

```diff
@@ -12,32 +12,81 @@ import { useConst } from './useConst';
  *
  * @example
  * ```ts
- * const observer = useCallback((nextValue) => {
+ * useObserver(source$, (nextValue) => {
  *   logger.log(nextValue);
- * }, []);
- * useObserver(source$, observer);
+ * });
  * ```
  */
 export function useObserver<T>(
   source$: Observable<T>,
-  observerOrNext: Partial<Observer<T>> | ((value: T) => void),
+  observerOrNext: PartialObserver<T> | ((value: T) => void),
```

Comment:
`Partial<Observer<T>>` is used to match to the argument type of `Observable.subscribe()`. `PartialObserver` type has extra `closed?: boolean` property.

Reply:
i see, replaced `PartialObserver<T>` to `Partial<Observer<T>>`